### PR TITLE
Fixed Gtk3 styling and import issues for packages in Stretch

### PR DESCRIPTION
**Pull request by radujipa**
This is to get TQ up and running on Stretch. The app should now look and perform just like the Jessie version.

### CODE REVIEW

**Comment by radujipa** [comment]
[(2) As a Stretch user, I want Terminal Quest to work](https://trello.com/c/38XHhYAD/1879-2-as-a-stretch-user-i-want-terminal-quest-to-work)

**Comment by radujipa** [comment]
This PR is now done

File: linux_story/gtk3/TerminalUi.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 # Terminal Gtk emulator
 
 
 import os
 
+from gi import require_version
+require_version('Gtk', '3.0')
+require_version('Vte', '2.90')
+
 from gi.repository import Vte, GLib, Gdk, Pango
```
Comment (skarbat) [inline linux_story/gtk3/TerminalUi.py]: Would it make sense to add `python-gi` dependency?

```
root@KXBN001:/# dpkg -S /usr/lib/python2.7/dist-packages/gi/overrides/Pango.py
python-gi: /usr/lib/python2.7/dist-packages/gi/overrides/Pango.py
```

File: bin/linux-story-gui [hunk 2]
```diff
@@ -24,9 +24,12 @@ Options:
 import os
 import sys
 import docopt
-from gi.repository import Gtk, GLib
 import Queue
 
+from gi import require_version
+require_version('Gtk', '3.0')
```
Comment (tombettany) [inline bin/linux-story-gui]: Do we not also need a `require_version` for `GLib`?

File: linux_story/gtk3/Storybook.py [hunk 3]
```diff
@@ -61,16 +61,13 @@ def __init__(self, width, height):
...
-            self.get_style_context().remove_class("normal")
+        style_context.remove_class("normal")
+        style_context.add_class("dark")
 
     def set_normal_theme(self):
         style_context = self.get_style_context()
-        if "dark" in style_context.list_classes():
-            self.get_style_context().remove_class("dark")
-            self.get_style_context().add_class("normal")
-            self.show_all()
```
[the remark this answers is not in the document]
Comment (radujipa) [reply linux_story/gtk3/Storybook.py]: That one was intended, yes. It's not needed and I think it was added in a Gtk confusion/argument @dev time.

**Comment by radujipa** [comment]
@tombettany I ended up renaming the old `tests` folder here since it wasn't finding "pytest` to run them. It was one of those _"How did this ever work?"_ moments. We'll need to rewrite & enable these at some other point.

File: bin/linux-story-gui (hunk 2 above)
Comment (tombettany) [reply bin/linux-story-gui]: I haven't but I just presumed that it was going to get grumpy about anything imported from `gi.repository`. No problems here then if you see no warning.

**Comment by tombettany** [comment]
@radujipa I had no idea that we even had tests in here.

**Pull request merged**